crates/kitsune_p2p/Makefile

```
@@ -0,0 +1,50 @@
+STATIC_CHECKS := static-fmt static-toml static-clippy static-doc
+
+.PHONY: $(STATIC_CHECKS) format build build-each test
+
+# Run all static checks
+check: $(STATIC_CHECKS)
+
+# Check formatting
+static-fmt:
+	cargo fmt --check
+
+# Check TOML formatting
+static-toml:
+	taplo format --check ./*.toml
+	taplo format --check ./crates/**/*.toml
+
+# Ensure our chosen style lints are followed
+static-clippy:
+	CHK_SQL_FMT=1 cargo clippy --all-targets -- --deny warnings
```

```suggestion
	cargo clippy --all-targets -- --deny warnings
```

There shouldn't be any sql in the kitsune workspace : )

---

Oops, yes :)